README.md

```diff
@@ -1,6 +1,6 @@
-# Cluster DNS Operator
+# DNS Operator
```

Comment:
How about "The DNS Operator" here and below?

Reply:
fixed